pkg/controller/storagecluster/initialization_reconciler.go

```diff
@@ -61,6 +61,23 @@ func (r *ReconcileStorageCluster) ensureStorageClasses(instance *ocsv1.StorageCl
 	return nil
 }
 
+func (r *ReconcileStorageCluster) newOBCStorageClass(initData *ocsv1.StorageCluster) *storagev1.StorageClass {
+	reclaimPolicy := corev1.PersistentVolumeReclaimRetain
+	retSC := &storagev1.StorageClass{
+		ObjectMeta: metav1.ObjectMeta{
+			Name: fmt.Sprintf("%s-ceph-retain-bucket", initData.Name),
```

Comment:
the retain or delete name should be based on `reclaimPolicy`.

Reply:
@leseb , from where will `reclaimPolicy` get that value? Currently `reclaimPolicy` is just a local variable and is not set from any outside condition.
From your sample SC from the converged mode doc,
```json
apiVersion: storage.k8s.io/v1
kind: StorageClass
metadata:
  name: rook-ceph-retain-bucket
provisioner: openshift-storage.ceph.rook.io/bucket
reclaimPolicy: Retain
parameters:
 objectStoreName: openshift-storage-object-store
 objectStoreNamespace: openshift-storage
 region: us-east-1
```
and it just sets the `reclaimPolicy` to `Retain`, which is the value of `corev1.PersistentVolumeReclaimRetain`.

Reply:
Ok, I got your point, please ignore the above comment...

Reply:
Done...